### PR TITLE
DH-1548 Remove unused sorts

### PR DESCRIPTION
**Pull request by yeahfro**
DH-1548

Change removes unused sorts for all entities. Details of removals are in the [JIRA ticket](https://uktrade.atlassian.net/browse/DH-1548).

Noted during this change, we do not have acceptance tests covering:
- investment project sorting
- interaction/service delivery sorting

### CODE REVIEW

File: test/acceptance/features/contacts/step_definitions/collection.js [hunk 1]
```diff
@@ -190,21 +180,14 @@ When(/^the contacts are sorted by (Country: A-Z)$/, async function (sortOption)
   await ContactList
     .section.firstContactInList
     .getText('@countryBadge', (result) => {
-      set(this.state, 'collection.firstItem.field', result.value.toLowerCase())
+      console.log(result)
```
Comment (ztolley) [inline test/acceptance/features/contacts/step_definitions/collection.js]: :trollface:

File: test/acceptance/features/events/step_definitions/collections.js [hunk 2]
```diff
@@ -103,7 +103,7 @@ Then(/^I filter the events list by start date/, async function () {
     .wait() // wait for xhr
 })
 
-Then(/^I sort the events list name A-Z$/, async function () {
+When(/^I sort the events list name A-Z$/, async function () {
```
Comment (teneightfive) [inline test/acceptance/features/events/step_definitions/collections.js]: Feels like these could be consolidated? They're very similar apart from the value.

File: src/apps/interactions/macros/collection-sort-form.js [hunk 3]
```diff
@@ -11,15 +11,11 @@ module.exports = {
       modifier: ['small', 'inline', 'light'],
       options: [
         { value: 'date:desc', label: 'Newest' },
-        { value: 'date:asc', label: 'Oldest' },
         { value: 'company.name:asc', label: 'Company: A-Z' },
-        { value: 'company.name:desc', label: 'Company: Z-A' },
         { value: 'contact.name:asc', label: 'Contact: A-Z' },
-        { value: 'contact.name:desc', label: 'Contact: Z-A' },
         { value: 'dit_adviser.name:asc', label: 'Adviser: A-Z' },
         { value: 'dit_adviser.name:desc', label: 'Adviser: Z-A' },
```
Comment (teneightfive) [inline src/apps/interactions/macros/collection-sort-form.js]: What about this one?

File: test/acceptance/features/contacts/step_definitions/collection.js (hunk 1 above)
Comment (feedmypixel) [reply test/acceptance/features/contacts/step_definitions/collection.js]: Sack him! :trollface:

Comment (ztolley) [reply test/acceptance/features/contacts/step_definitions/collection.js]: 😿

File: test/acceptance/features/events/step_definitions/collections.js (hunk 2 above)
Comment (ztolley) [reply test/acceptance/features/events/step_definitions/collections.js]: I agree. There's a few of the collection steps that can be consolidated, not just this one. I've added to the list of things to consider for tech debt to address this, but not in this PR as the main focus needs to be to get rid of unused sort options.

File: src/apps/interactions/macros/collection-sort-form.js (hunk 3 above)
Comment (ztolley) [reply src/apps/interactions/macros/collection-sort-form.js]: What do you mean? You are asking why it is being removed? If so then all of the users said they don't want it though they do want A-Z. As odd as it sounds the team decided to not always provide an inverse filter.

**Pull request merged**